### PR TITLE
Enables some unit tests on travis.

### PR DESCRIPTION
**Pull request by yurydelendik**
Implements idea from #7179 about reusing the same tests and run portion of them on Travis CI via node.

### CODE REVIEW

**Review by Snuffleupagus — APPROVED** [review]
Nice to be able to run unit-tests on Travis too!
Looks good to me; with one question below.

If I understand correctly, the only unit-tests that we (currently) cannot run on Travis are ones that load various files themselves (e.g. PDFs, CMAPs, etc.).
However, since it seems like a good idea to run as many of the unit-tests on as possible: Should we also add the `metadata_spec.js`, `ui_utils_spec.js` and `util_spec.js` tests to the ` test/unit/clitests.json` manifest file?

**Comment by yurydelendik** [comment]
> Should we also add the metadata_spec.js, ui_utils_spec.js and util_spec.js tests to the test/unit/clitests.json manifest file?

I missed  util_spec.js by accident. metadata_spec.js and ui_utils_spec.js (as well as cmap, annotation_layer and api) require DOM objects such as XMLHttpRequest or document.

File: test/unit/clitests.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "spec_dir": "test/unit",
+  "spec_files": [
+     "cff_parser_spec.js",
```
Comment (timvandermeij) [inline test/unit/clitests.json]: Nit: this line and the ones below are indented with three spaces instead of two.

Comment (yurydelendik) [inline test/unit/clitests.json]: fixed

**Review by timvandermeij — APPROVED** [review]

**Comment by timvandermeij** [comment]
Awesome, thanks!

**Pull request merged**